File: Modul-4/Soal-2/PRAK402-2410817220012-NOOR-KHALISA.py
batas = int(input("Input: "))

ganjil = 1
while ganjil <= batas:
    print(ganjil, end=" ")
    ganjil += 2
print()

genap = batas if batas % 2 == 0 else batas - 1
while genap >= 2:
    print(genap, end=" ")
    genap -= 2
print()

File: Modul-4/Soal-2/PRAK402-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(){
    int batas;

    printf("Input: ");
    scanf("%d", &batas);
    int ganjil = 1;
    while (ganjil <= batas)
    {
       printf("%d ", ganjil);
       ganjil += 2;
    }printf("\n");

    int genap = (batas % 2 == 0) ? batas : batas -1 ;
    while (genap >= 2)
    {
        printf("%d ", genap);
        genap -= 2;
    }printf("\n");

    return 0;
}